deployments.json

```diff
@@ -836,6 +836,37 @@
             "contract": "MorphoBlueHyperdrive",
             "address": "0xd41225855A5c5Ba1C672CcF4d72D1822a5686d30",
             "timestamp": "2024-08-08T15:43:26.033Z"
+        },
```

Comment:
just fyi this is here because I forgot to commit/push it after deploying last week